i18n/json_to_js.js

```diff
@@ -0,0 +1,49 @@
+const async = require('async');
+const fs = require('fs');
+const path = require('path');
+const glob = require('glob');
+
+// Globals
+const PATH_INPUT = path.resolve(__dirname, '../msg/json/*.json');
```

Comment:
We could pull directly from Transifex, but if we let scratch-l10n manage the pull it can also run tests to check that translators didn't mess up the json. If scratch-l10n exports the .js files, maybe scratch-blocks can just handle things in webpack.

One thing to note is that developers need transifex credentials to push/pull translations.